src/index.js

```diff
@@ -31,6 +31,7 @@ export default async (
     const isValidLink =
       node.type === 'link' &&
       node.title === null &&
+      node.children[0] &&
```

Comment:
I think this makes it a bit clearer

```suggestion
      node.children.length === 1 &&
```

Reply:
done